build.sbt

```diff
@@ -201,8 +202,8 @@ lazy val `quill-async-postgres` =
     )
     .dependsOn(`quill-async` % "compile->compile;test->test")
 
-lazy val `quill-cassandra` =
-  (project in file("quill-cassandra"))
+lazy val `quill-cassandra-core` =
```

Comment:
This would make the lagom module also depend on the datastax driver. Let's have the following modules:

1. `quill-cassandra`, which is generic and doesn't have any driver dependency
2. `quill-cassandra-datastax` that depends on `quill-cassandra` and uses the datastax driver
3. `quill-cassandra-lagom` that depends on `quill-cassandra` and uses lagom

Please move as many tests as possible to `quill-cassandra`

Reply:
Hi, I've tried to seperate it. But it seems both `quill-cassandra-datastax` and `quill-cassandra-lagom` will depend on the datastax driver. And It's really hard to seperate it from the common module.